docs/source/get_started/new_project.md

```diff
@@ -112,45 +104,24 @@ This command automatically opens a browser tab to serve the visualisation at `ht
 
-## Where next?
-You have completed the section on Kedro project creation for new users. Now choose how to learn more:
-
-* Understand more about Kedro: The following page explains the [fundamental Kedro concepts](./kedro_concepts.md).
-
-* Learn hands-on: If you prefer to learn hands-on, move on to the [spaceflights tutorial](../tutorial/spaceflights_tutorial.md). The tutorial illustrates how to set up a working project, add dependencies, create nodes, register pipelines, set up the Data Catalog, add documentation, and package the project.
-
-* How-to guide for notebook users: The documentation section following the tutorial explains [how to combine Kedro with a Jupyter notebook](../notebooks_and_ipython/kedro_and_notebooks.md).
-
-If you've worked through the documentation listed and are unsure where to go next, review the [Kedro repositories on GitHub](https://github.com/kedro-org) and [Kedro's Slack channels](https://slack.kedro.org).
-
-
-## More information about the `pandas-iris` example project
-
-If you used the `pandas-iris` starter to create an example project, the rest of this page gives further information.
-
-<details>
-<summary>Expand for more details.</summary>
+## Summary
 
-### Background information
-The Iris dataset was generated in 1936 by the British statistician and biologist Ronald Fisher. The dataset contains 150 samples, comprising 50 each of 3 different species of Iris plant (*Iris Setosa*, *Iris Versicolour* and *Iris Virginica*). For each sample, the flower measurements are recorded for the sepal length, sepal width, petal length and petal width.
+After you have [set up Kedro](install.md), you can use `kedro new` to create a basic Kedro project containing project directories and basic code, which you can configure depending on the tooling and example code you need.
 
-![](../meta/images/iris_measurements.png)
+Alternatively, you can use `kedro new` and [pass in a configuration file](#create-a-basic-project-from-a-configuration-file) to manually control project details such as the name, folder and package name.
 
-A machine learning model can use the Iris dataset to illustrate classification (a method used to determine the type of an object by comparison with similar objects that have previously been categorised). Once trained on known data, the machine learning model can make a predictive classification by comparing a test object to the output of its training data.
+After you create a new project you need to navigate to its project folder and install its dependencies: `pip install -r requirements.txt`
 
-The Kedro starter contains a single [pipeline](../resources/glossary.md#pipeline) comprising three [nodes](../resources/glossary.md#node) responsible for splitting the data into training and testing samples, running a 1-nearest neighbour classifier algorithm to make predictions and accuracy-reporting.
+* **To run the project**: `kedro run`
+* **To visualise the project**: `kedro viz`
```

Comment:
Very soon `kedro viz run` or variations (nothing to do for now I guess)

Reply:
Yes, a global search and replace when the time comes.